second-task/src/Components/Pagination.tsx

```diff
@@ -0,0 +1,37 @@
+import { FC } from "react";
+import Page from "./Page";
+
+export interface PaginationProps {
+  activePage?: number;
+  totalItems: number;
+  perPage: number;
+  withActions?: boolean;
+  classes?: {
+    btn?: string;
+    activeBtn?: string;
+  };
+  onChangePage: (newPage: number) => void;
+}
+
+const Pagination: FC<PaginationProps> = ({ activePage, perPage, totalItems, onChangePage, classes, withActions }) => {
+  const realActivePage: number = activePage !== undefined ? activePage : 1;
```

Comment:
first of all it can be written as 
`const realActivePage: number = activePage || 1;`
but better is to set default prop
<img width="437" alt="image" src="https://user-images.githubusercontent.com/114758808/193307370-11ee8174-3cff-4528-8509-7c8fb277d0ea.png">
in this case x and y will be set to 10 and 20 if they are undefined